lgtm.yml

```diff
@@ -6,7 +6,7 @@ extraction:
         configure: # Customizable step used only by C/C++ extraction.
             command:
                 - source scripts/bootstrap.sh && gn gen out/custom
-                  --args='target_os="linux_x64"'
+                  --args='target_os="linux"'
```

Comment:
You can leave target_os blank.

Reply:
Would that make it host_os?
i.e. should I just drop the the --args parameter?

Reply:
I dropped the args for now.

Reply:
Yes, the default is host_os so dropping args completely is equivalent.